client/app/bundles/course/assessment/submission/components/answers/Programming/index.jsx

```diff
@@ -1,130 +1,25 @@
-import { useRef, useState } from 'react';
 import { useFieldArray, useFormContext, useWatch } from 'react-hook-form';
-import { defineMessages } from 'react-intl';
-import { Close, ThumbDown, ThumbUp } from '@mui/icons-material';
-import {
-  Box,
-  Card,
-  CardActions,
-  CardContent,
-  Drawer,
-  IconButton,
-  Tooltip,
-  Typography,
-} from '@mui/material';
-import { green, grey, orange, red, yellow } from '@mui/material/colors';
+import { Box } from '@mui/material';
```

Comment:
https://github.com/Coursemology/coursemology2/blob/master/client/README.md#-do-not-use-muis-box-component